src/govuk/components/input/template.test.js

```diff
@@ -382,4 +389,107 @@ describe('Input', () => {
       expect($component.attr('inputmode')).toEqual('decimal')
     })
   })
+
+  describe('when it includes a prefix', () => {
+    it('renders the input wrapper', () => {
+      const $ = render('input', examples['with prefix'])
+
+      const $wrapper = $('.govuk-form-group > .govuk-input__wrapper')
+      expect($wrapper.length).toBeTruthy()
+    })
```

Comment:
Minor, but the pattern in this file (and I've seen in other test files) is to leave a new line between each test